internal/providers/terraform/azure/app_service_certificate_order.go

```diff
@@ -16,10 +15,7 @@ func GetAzureRMAppServiceCertificateOrderRegistryItem() *schema.RegistryItem {
 }
 
 func NewAzureRMAppServiceCertificateOrder(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
-	location := "Global"
-	if strings.HasPrefix(location, "usgov") {
-		location = "US Gov"
-	}
```

Comment:
we need this as for some reason Azure likes to charge the US gov more for the same service, i'll fix it.